config/vendor-api-0.8.0.yml

```diff
@@ -597,7 +597,7 @@ components:
           example: Julia was my mentor at Cheadle Hulme High
         reference:
           type: string
-          description: "Optional. The reference content provided by the referee, once it is available"
+          description: "Required to send the application to the provider. The reference content provided by the referee, once it is available"
```

Comment:
Do we need this qualifier at all (or the `once is it available` clause) if providers will only see responses with references complete?